### ISSUE TITLE
Problems creating drill down Function

### ISSUE THREAD

**Issue by pgksunilkumar** [google/google-visualization-issues#2378]
no matter where i click on the donut pie chart i see for values changes can some one tell me why this happens

example i am attaching the file
[piechart.txt](https://github.com/google/google-visualization-issues/files/550338/piechart.txt)

**Comment by nbering** [comment]
Can you put your example in jsFiddle? I'm willing to take a look for you but I don't download people's random files from github issues.

**Comment by pgksunilkumar** [comment]
http://jsfiddle.net/pgksunilkumar/g6axn7qb/1/

**Comment by nbering** [comment]
Can you rephrase the question? I don't understand what the problem is. As far as I can follow the intent of your code it seems to work fine. What is the problem?

**Comment by pgksunilkumar** [comment]
when i click on Category A it takes to take me into sub categories and works fine however when i click on any one of the sub category it shows some other sub categories 

when i click on back button it will reload the entire page with the graph which is not right way i just want the graph to updated
